src/backend/marsha/core/xapi.py

```diff
@@ -140,13 +140,13 @@ def from_website(self, document, statement, current_site, user):
             document, statement, homepage=current_site.domain, user=user
         )
 
-    def from_lti(self, document, statement, jwt_token):
+    def from_lti(self, document, statement, jwt_token, domain):
```

Comment:
WDYT about having an optional `domain` parameter, with `document.playlist.consumer_site.domain` as a default ? I feel like it might simplify the usage.